blueocean-dashboard/src/main/js/Dashboard.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { Component, PropTypes } from 'react';
+
+const { object, node } = PropTypes;
+
+class Dashboard extends Component {
+
+    getChildContext() {
+        const {
+            params,
+            location,
+        } = this.props;
+
+        return {
+            params,
+            location,
+        };
+    }
+
+    render() {
+        return this.props.children; // Set by router
+    }
+}
+
+Dashboard.propTypes = {
+    params: object, // From react-router
+    children: node, // From react-router
+    location: object, // From react-router
+};
+
+Dashboard.childContextTypes = {
+    params: object, // From react-router
+    location: object, // From react-router
+};
+
+export default Dashboard;
```

Comment:
I refactored some of the truly global things like params and location out of OrganizationPipelines because as we add more routes, odds are the may not descend from OrgPipelines but we still want these items to be available in all contexts.